### ISSUE TITLE
直接python tools/train.py 并不能找到yolox.core这些module

### ISSUE THREAD

**Issue by wuzhuoyuan** [Megvii-BaseDetection/YOLOX#737]
在readme中的Quick Start命令行，由于在yolox根目录运行tools下的train.py，sys path没有yolox目录下的module。ModuleNotFoundError: No module named 'yolox'

**Comment by cheng-orange** [comment]
修改tools目录下的demo.py,在import torch后加上下面两句命令。

import sys
sys.path.append(r'根目录')

**Comment by quanh1990** [comment]
没有安装吧？

python setup.py develop 之后python就知道yolox包在哪儿了

**Comment by FateScript** [comment]
Yes, please install yolox first. Make sure ```import yolox``` works everywhere.
